File: newcoder_pat_basic/1023/go/main.go
package main

import "fmt"

func main() {
	var broken [129]bool
	var c byte
	for ; c != '\n'; {
		fmt.Scanf("%c", &c)
		broken[(int)(c)] = true
		if c >= 'A' && c <= 'Z' {
			broken[c+32] = true
		}
	}
	for c = 0; c != '\n'; {
		fmt.Scanf("%c", &c)
		if broken[c] || (c >= 'A' && c <= 'Z' && broken[(int)('+')]) {
			continue
		}
		fmt.Printf("%c", c)
	}
	fmt.Printf("\n")
}

File: newcoder_pat_basic/1023/c/main.c
#include <stdio.h>

int main() {
  char broken[129] = {0}, c;
  do {
    scanf("%c", &c);
    broken[(int)(c)] = 1;
    if(c >= 'A' && c <= 'Z') broken[c+32] = 1;
  } while(c != '\n');
  do {
    scanf("%c", &c);
    if(broken[(int)(c)] == 1) continue;
    if(c >= 'A' && c <= 'Z' && broken[(int)('+')] == 1) continue;
    printf("%c", c);
  } while(c != '\n');
  printf("\n");
  return 0;
}
